README.md

```diff
@@ -18,59 +29,59 @@ If you're new here, here are some things you should know:
+### Install Ruby
+You will need to first install the required ruby version specified in the `.ruby-version` file.  It is highly recommended you use a ruby version manager like [rbenv](https://github.com/rbenv/rbenv) or [asdf](https://asdf-vm.com/)
 
-For a general overview of the application, please see the [Application Overview](https://github.com/rubyforgood/human-essentials/wiki/Application-Overview) wiki article.
+GoRails has a very detail oriented guide for installing ruby on Ubuntu, Windows, and macOSX. You can check out that guide [here](https://gorails.com/setup/osx/12-monterey). **Follow only the Installing Ruby step, as our project setup differs**
 
-### Origins
+Once you've succesfully installed ruby. You may proceed to the next section!
 
+### Install Node
+You must install the version of node specified in `.nvmrc`. It is highly recommended to use [nvm](https://github.com/nvm-sh/nvm).
 
-This project took what we built for the [Portland Diaper Bank in 2016](https://github.com/rubyforgood/pdx_diaper) and turned it into a multitenant application, something that all essentials banks can use. We re-used models, code and other documentation where applicable as well as implemented new features and functionality requested by the prime stakeholder (PDXDB). We're super excited to have had Rachel Alston, the director of the Portland Diaper Bank, attending our event in 2017, providing guidance and giving us the best chance of success!
+You can follow the guide [here](https://heynode.com/tutorial/install-nodejs-locally-nvm/) for setting up node.
 
+*NOTE: It's possible that Node version 12 may cause you problems, see issue #751. Node 10 or 11 seem to be fine.*
 
-## Development
+### Install Postgres
+You must install postgres and run the database locally. Instructions differ depending on the operating system.
 
-### Installation Instructions
+Follow one of these guides to install postgres:
 
-The `installation.md` file ([https://github.com/rubyforgood/human-essentials/blob/main/installation.md](https://github.com/rubyforgood/human-essentials/blob/main/installation.md)) has detailed instructions for installation and configuration of an Ubuntu host to run this software. Although there is not a document for Mac OS, it may be helpful for that as well.
+###### MacOSX
+https://www.digitalocean.com/community/tutorials/how-to-use-postgresql-with-your-ruby-on-rails-application-on-macos
 
-### Ruby Version
-This app uses Ruby version 3.0.2, indicated in `/.ruby-version` and `Gemfile`, which will be auto-selected if you use a Ruby versioning manager like `rvm`, `rbenv`, or `asdf`.
+###### Ubuntu
+TODO
```

Comment:
Don't we have ubuntu instructions here: https://github.com/rubyforgood/human-essentials/blob/main/installation.md

Or are they out of date?

Reply:
@seanmarcia I'am not sure. We do have a Ubuntu user that I can double check with.

Reply:
Hey, I'm the Ubuntu user ;)

The setup is basically working and up-to-date, just got a few remarks:

- I did not install the packages mentioned in the beginning, as I already had them installed.
- Running sudo npm -g install yarn did not work ("command not found"), just npm -g install yarn did the trick.
- I skipped over the rvm part, as I am using rbenv.  I'd probably put a message there for people to double-check, whether they already have another version manager installed, as for example rbenv and rvm _will_ clash with one-another.
- The part where you explain how to set up the environment variables wasn't really clear to me. I just added them to my root .bashrc in the end and set it as source inside the project folder, which worked just fine.
- Running rails db:setup db:migrate db:test:prepare db:seed did cause an issue for the first time around during DB seeding. Something about "address1" not being a valid Active Model attribute or something. Had to restart my machine due to something unrelated, though, and the next time around the command ran just fine and I am no longer able to reproduce the problem now.

Project seems to be running just fine on my Ubuntu machine now and the setup went pretty quick all in all.

Reply:
Thanks for the feedback @florianfelsing! I had a follow up question for:
> The part where you explain how to set up the environment variables wasn't really clear to me. I just added them to my root .bashrc in the end and set it as source inside the project folder, which worked just fine.

The goal was to allow the `bin/setup` command to setup your .env file which would be used. Was that unclear and how could we be clearer about that?

Reply:
> Thanks for the feedback @florianfelsing! I had a follow up question for:
> 
> > The part where you explain how to set up the environment variables wasn't really clear to me. I just added them to my root .bashrc in the end and set it as source inside the project folder, which worked just fine.
> 
> The goal was to allow the `bin/setup` command to setup your .env file which would be used. Was that unclear and how could we be clearer about that?

I overlooked that part of the process since it wasn't mentioned in the Ubuntu-specific installation guide. Only realized too late that it was to be found in the README.

Also as for what @divagueame wrote, this is exactly the error that I encountered once and that disappeared once I had to reboot my system.